ci-operator/jobs/infra-image-mirroring.yaml

```diff
@@ -1247,3 +1247,51 @@ periodics:
     - configMap:
         name: image-mirror-mappings
       name: config
+- agent: kubernetes
+  cluster: app.ci
+  cron: '@hourly'
```

Comment:
How important this is it to run every hour?

Reply:
Not very critical. However the idea was that we can be quicker than we used to be with the mirroring into the openshift organization when it took 4,5,6 hours (OCP release and test).
The time gained here would allow us to use the operand fixes in the operator repository faster. This comes quite touchy when some Bugs needs to be fixed fast or when the release deadline is coming.